### DISCUSSION TITLE
In logging.log_filename_template -- "Update by 3.0" -- Why?

### DISCUSSION THREAD

**Discussion by garywhiteford** [apache/airflow#34376] [Q&A] [answered]
# "Please update your config before Apache Airflow 3.0"
```
./airflow2.3/lib/python3.8/site-packages/airflow/configuration.py:412:
FutureWarning: The 'log_filename_template' setting in [logging] has the old default value of '{{ ti.dag_id }}/{{ ti.task_id }}/{{ ts }}/{{ try_number }}.log'. This value has been changed to 'dag_id={{ ti.dag_id }}/run_id={{ ti.run_id }}/task_id={{ ti.task_id }}/{% if ti.map_index >= 0 %}map_index={{ ti.map_index }}/{% endif %}attempt={{ try_number }}.log' in the running config, but please update your config before Apache Airflow 3.0.
```

## Questions
* Why? 
* What is going to change? 
* What happens if I don't? 
* Where can I listen or read more about this? 

## Reference
https://github.com/apache/airflow/blob/013c95bfe6d2b4de44633ab1a0c326cd0969fcf3/airflow/configuration.py#L439C14-L443

**Comment by Taragolis** [comment, marked as answer]
> Why?
>Where can I listen or read more about this?

[See Airflow 2.3.0 changes ](https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#default-templates-for-log-filenames-and-elasticsearch-log-id-changed-21734)

> What is going to change?
> What happens if I don't?

Format defined in `[logging] log_filename_template`  will be ignored and I guess something bad happen. Nobody know **when** or **if** Airflow 3.0 released and which additional changes happen.

**Comment by garywhiteford** [comment]
Perfect... that's my answer! Thanks also for the pointers to the relase notes. Don't know why I didn't think of that. 

So it is optional and can be configured.... differently. 

Now, is there a way to suppress the warning? 

![image](https://github.com/apache/airflow/assets/33525443/72f1551e-db2a-4d9d-9968-0cf29ce4cc3b)

**Reply by Taragolis** [reply]
I you happy with new format, for example how it looks like

![image](https://github.com/apache/airflow/assets/3998685/5f55c385-9fbf-4684-a713-7dff23f80f7a)

You just need to remove your current value from `[logging] log_filename_template`

Otherwise you should replace to something other then `{{ ti.dag_id }}/{{ ti.task_id }}/{{ ts }}/{{ try_number }}.log`
